_config/api.yml

```diff
@@ -0,0 +1,18 @@
+---
+Name: appapi
+After:
+  - '#graphqlconfig'
+---
+
+SilverStripe\Control\Director:
+  rules:
+    'graphql': '%$SilverStripe\GraphQL\Controller.default'
```

Comment:
why change the GraphQL server to the default one?

Reply:
This might be ok, just need to get a bit more of an understanding about the purpose of custom servers, when/why they should be used.